### DISCUSSION TITLE
Love this feature! the task completion is VERY slow though

### DISCUSSION THREAD

**Discussion by tallguyjenks** [community/community#4338] [Projects and Issues]
Loving all the new beta releases recently! lots to play with and implement!

Im very big on outlining and task boxes so I tend to make a lot of them. When checking off several tasks in quick succession, after ticking a checkbox all the other tasks in the issue become immutable for some background process to complete that updates the UI for the issues "1/3 complete" status or something.

The slow completion of tasks and inability to tick several of them in quick succession is a point of friction for me :)

That said, the "Convert to issue" feature though 🤯️

**Comment by ElijahLynn** [comment]
I was/am working on an extension to allow for editing tasks in the UI and was digging through the code and found that it is because every time you drag or toggle a checkbox it makes a network request to the server to POST the data to GitHub and save it. Every single operation is a save event. It is very irritating and makes task management non-ideal.

**Comment by gordonel** [comment]
+1 here. I needed to switch from Notion to GitHub issues for one of my projects, and Notion's checkboxes had instant response
